packages/unmock-core/src/index.ts

```diff
@@ -71,21 +63,43 @@ export abstract class CorePackage implements IUnmockPackage {
     this.backend.reset();
   }
 
-  public get allowedHosts() {
+  // Allowd Hosts methods
+  public setAllowedHosts(urls: Array<string | RegExp> | string | RegExp): void {
```

Comment:
Could also wrap these in a `allowedHosts` object, with `allowedHosts.add` and other methods to reduce the clutter at top-level.